CUSTOM_CELLS.md

```diff
@@ -0,0 +1,76 @@
+# MessageKit Custom Cell Guide(s)
+
+- [How can I add a custom cell?](#how-can-i-add-a-custom-cell)
+
+## How can I add a custom cell?
+
+**Note:** If you choose to use the `.custom` kind you are responsible for all of the cell's layout. Any `UICollectionViewCell` can be returned for custom cells which means any of the styling you provide from the `MessageDisplayDelegate` will not effect your custom cell. Even if you subclass your cell from `MessageContentCell`.
```

Comment:
`effect` -> `affect`

Can a custom cell subclass, such as a subclass of `MessageContentCell` or `TextMessageCell`, still access the `MessageDisplayDelegate` in some way? So that the framework user could still forward specific calls to generic delegate implementations? Just a thought.